tests/conftest.py

```diff
@@ -23,13 +23,13 @@ def participant_metrics_sample():
     return json.dumps(
         [
             {
-                "measurement": "CPU utilization",
+                "measurement": "participant",
                 "time": 1234326435,
                 "tags": {"id": "127.0.0.1:1345"},
                 "fields": {"CPU_1": 90.8, "CPU_2": 90, "CPU_3": "23", "CPU_4": 0.00,},
             },
             {
-                "measurement": "CPU utilization",
+                "measurement": "participant",
```

Comment:
Maybe would be nice to have a separate field for name of the sender, like:
```
"measurement": "CPU utilization",
"sender": "participant,
```

Reply:
The `id` of the participant is already stored in `tags` (https://github.com/xainag/xain-sdk/blob/development/xain_sdk/participant.py#L269) Did I get it right or do you mean something else with `sender`?

Reply:
the naming is a bit misleading, but i am ok with it